specification/liftrastronomer/resource-manager/Astronomer.Astro/stable/2023-08-01/examples/Organizations_CreateOrUpdate_MaximumSet_Gen.json

```diff
@@ -100,7 +100,7 @@
         },
         "tags": {},
         "location": "mhqthlsatwvqkl",
-        "id": "bhslekyvgkfomahtvjiin",
+        "id": "/subscriptions/cbdb67be-e103-4770-a797-2a2fa42eb6f3/resourceGroups/npachori-rg/providers/Astronomer.Astro/organizations/astr300124",
```

Comment:
>npachori-rg [](http://example.com/codeflow?start=82&length=11)

Can we make this generic instead of an alias/name?

Reply:
done